### ISSUE TITLE
Write microcopy for new "Latest updates" pieces

### ISSUE THREAD

**Issue by noahmanger** [18F/fec-cms#672]
This is a small microcopy task for adding descriptions of for the remaining items that will be added to the Latest Updates. 

**Completion criteria:**
Write descriptions for:
- [ ] FEC Record articles
- [ ] Meeting agendas
- [ ] Sunshine notices
- [ ] Tips for Treasurers

**Comment by emileighoutlaw** [comment]
[Content draft](https://docs.google.com/document/d/1_7LZD0CHr_p-LSYlZKbX6U14lx6dt70lezuBIkwdkbQ/edit#) 🔒  is ready for FEC review. It includes introductory content and drop-down content.


#### Actually writing this raised a few questions (of course :upside_down_face:):

1. (cc @noahmanger for this) Even with one intro sentence per content type, we end up with a ton of expository text that explains what people can find on the page. I think it's way too much for readers. I'd like to try cutting everything except the first sentence  (_The FEC regularly publishes....._). What do you think? 

2. (cc @noahmanger and @nickykrause for memory confirmation) We are planning to include Sunshine Act Notices and meeting agendas in `Latest updates`. But we _aren't_ including:
    - Draft meeting minutes
    - Approved meeting minutes
    - Late-added agenda item requests (usually PDF documents)
    - Podcasts of open meetings

Is that correct to your recollection, Nicky? I ask because: If they don't go on this page, I believe they will only exist on the calendar? That seems like a fine IA decision, but I don't think we've actually made a plan in our quarterly planning for getting those historical documents linked from the calendar yet?

**Comment by noahmanger** [comment]
So, I think that some expository content is important. I get that it's a lot, but I wonder if a solution might be to use an accordion? Given that there's lots of things on this page I think it's important to say _somewhere_ what those things are and who they're for.

You're right about the additional meeting documents, and we really haven't come up with a plan for these. I'm adding to my running list of things we might do next quarter.

**Comment by nickykrause** [comment]
@emileighoutlaw: The inclusion of Sunshine Act Notices and Meeting Agendas within `Latest Updates` sounds correct. However, I don't remember any specific discussion of the four sub-items you listed (Draft meeting minutes, Approved meeting minutes, Late-added agenda item requests, and Podcasts of open meetings). If these are only available via the calendar item itself, I think that's fine. 

Also, in case it is helpful, here is a link to the issue where this all was discussed: https://github.com/18F/fec-cms/issues/524

**Comment by emileighoutlaw** [comment]
(Thanks for that issue number, Nicky!)

Re: the content. My concern about expository content (content that explains what people are looking at) is that in this case, there's just a ton of it — it's just a lot to ask people to read. Additionally, in general, expository content can come off a little condescending to experienced users. 

So thinking about this challenge a different way, it seems like what we want to solve for is:
- A way to orient the new folks to what they're looking at, and at the same time
- Not making the experienced folks feel like we're talking down to them. 

It occurs to me that this might actually be a really good place for tooltips. We use this pattern already on the calendar for almost the exact same purpose:

<img width="821" alt="screen shot 2017-01-06 at 12 04 55 pm" src="https://cloud.githubusercontent.com/assets/10552702/21727777/860d1eac-d408-11e6-92a5-55ff5694d5fb.png">

So we'd be staying consistent with our style if we added those here:

<img width="988" alt="screen shot 2017-01-06 at 12 04 39 pm" src="https://cloud.githubusercontent.com/assets/10552702/21727871/02981936-d409-11e6-84e9-10523535e3f2.png">

As a second, additional clue, I'd also feel good about adding publication specific info to the top of the page when we're filtering to specific content types. So from the main landing, the content would look like:
<img width="897" alt="screen shot 2017-01-06 at 12 11 19 pm" src="https://cloud.githubusercontent.com/assets/10552702/21728020/a97a4076-d409-11e6-9d7b-bba992fecb6c.png">



But when filtered down to only press releases, we get that additional bit of orienting info (and since it's only a little bit, it's less overwhelming and comes off tonally more appropriate, in my opinion):
<img width="941" alt="screen shot 2017-01-06 at 12 12 27 pm" src="https://cloud.githubusercontent.com/assets/10552702/21728039/bc10fef0-d409-11e6-999f-1094e7368bd9.png">


What do you think, @noahmanger?

**Comment by noahmanger** [comment]
I get your point about lengthy expository content and seems worth avoiding. I can't quite put my finger on why this solution feels off, but it does. I think it's because I think it's valuable to see a listing of all the different update types and what they are all at once in order to know what to refine to. 

I have two additional ideas that might solve both issues.

First, use our accordion pattern:

![accordion 1](https://cloud.githubusercontent.com/assets/1696495/21787128/4c1f4776-d67c-11e6-8222-dfaf0c988461.png)

Second, something like this:
![accordion 2](https://cloud.githubusercontent.com/assets/1696495/21787136/5c4c9be4-d67c-11e6-8700-12f1a8ac021c.png)

Thoughts?

**Comment by emileighoutlaw** [comment]
Of the two options, I think I prefer the first option, the accordion pattern. But I could be convinced to go with either one if you have a strong preference?

**Comment by noahmanger** [comment]
That works for me. @jenniferthibault & @nickykrause ?

**Comment by nickykrause** [comment]
Of the two, I also prefer the accordion. Couldn't hurt to try it, I think!

**Comment by jenniferthibault** [comment]
If you'd like to use an accordion here, I'd recommend that we make it as wide as the text-width, instead of container width. 

![screen shot 2017-01-11 at 2 41 56 pm](https://cloud.githubusercontent.com/assets/11636908/21863605/2486641a-d80c-11e6-95c1-8a0af59f9afd.png)

With a single-panel accordion, it's risky to have it so wide because it's less likely to visually register as a functioning component because the button is so far to the side, and they're so narrow. (We saw this in usability testing the data landing page charts when we used an accordion for "About this data" underneath)

The closest we've come to this in existing patterns on the site is on committee pages, but they're XL height and have multiple panels, not just one.

<img width="1016" alt="screen shot 2017-01-11 at 2 26 30 pm" src="https://cloud.githubusercontent.com/assets/11636908/21863088/02eb87c4-d80a-11e6-93e9-26f11c261de6.png">

The "show more" would be a new pattern, but it acts much the same as an accordion but within a much smaller space, which seems like a positive to me. I'm not opposed to this at all, but glad to try the accordion first.

I'm hitting a larger experience snag for myself on the content inside the accordion than the element itself though. The activation text is "About latest updates" but the text inside tells me about audience segments, but doesn't really connect it to why we group docs by people.

I don't want to solutioneer over @emileighoutlaw too much, but also don't want to offer unconstructive feedback, so one possible way of handling this might be to add a **For** in front of each category? For example:
> "For media professionals" 
> "For candidates and committees"
> "For members of the public"

If not, maybe just back to definitions of each content type being the header with its own paragraph?

**Comment by jenniferthibault** [comment]
Sorry, you got all the thoughts at once because I hadn't been following this issue until tagged in!

**Comment by emileighoutlaw** [comment]
Not solutioneering over me at all Jen! I hadn't gotten super detailed in my review of the content yet (I left things a little unrefined when I wasn't sure I wanted content here at all). 

In terms of design, I don't have strong feelings between the accordion pattern and the `Show more +` pattern— my big preference is that we don't bombarding people with expositional content they have to scroll past or ignore to get to the actual meat of the page. 

**tl;dr:** I'm going to put some refined content below for each pattern, in case that helps us make an informed decision. Based on Jen's notes above, I find I'm leaning toward the "Show more +" option, if we think it makes a clearer interaction.

## Accordion option
Content: 

> ### Latest updates
> Search or browse the latest information about the Commission and federal campaign finance law.

>  ##### About latest updates  ............................... +
> The FEC regularly publishes information for media professionals, for candidates and committees and for members of the public.

>  **For media professionals**
> Press releases are published as news happens. Weekly Digests are published every Friday and summarize the week's publicly disclosed activity.

> **For candidates and committees**
> FEC Record articles are published as news happens. Tips for Treasurers are published once a week. 

> **For members of the public**
> Sunshine Act Notices announce open meetings and executive sessions. Notices of open meetings are published at least a week before a meeting and announce the date, time and a general list of items for discussion. Meeting agendas are published XXXX. 


## "Show more" option
> ### Latest updates
> Search or browse the latest information about the Commission and federal campaign finance law.

> ##### Show more +
> The FEC regularly publishes information for media professionals, for candidates and committees and for members of the public.

>  **For media professionals**
> Press releases are published as news happens. Weekly Digests are published every Friday and summarize the week's publicly disclosed activity.

> **For candidates and committees**
> FEC Record articles are published as news happens. Tips for Treasurers are published once a week. 

> **For members of the public**
> Sunshine Act Notices announce open meetings and executive sessions. Notices of open meetings are published at least a week before a meeting and announce the date, time and a general list of items for discussion. Meeting agendas are published XXXX.

**Comment by noahmanger** [comment]
Let's go with the accordion for now since it's an established pattern. If we end up wanting to make a new one that we use elsewhere, we can replace it.

**Comment by emileighoutlaw** [comment]
Content shouldn't change either way, so we should be good to go. I'd like @AmyKort  to give this a final review before we go live. [Updated content for this page](https://docs.google.com/document/d/1_7LZD0CHr_p-LSYlZKbX6U14lx6dt70lezuBIkwdkbQ/edit#) 🔒:


> ## Latest updates
> Search or browse the latest information about the Commission and federal campaign finance law.

> ##### About latest updates +
> The FEC regularly publishes information for media professionals, members of the public and candidates and committees.

> **For media professionals**
> Weekly Digests are published every Friday and summarize the week's publicly disclosed activity. Press releases are published as news happens. 

> **For candidates and committees**
> FEC Record articles are published as news happens. Tips for Treasurers are published once a week.

> **For members of the public**
> Meeting agendas are usually published a week before a scheduled meeting. 

> Sunshine Act Notices are announcements of open meetings and executive sessions— notices for open meetings are published at least a week before a scheduled meeting and announce the date, time and a general list of items for discussion.

**Comment by AmyKort** [comment]
Two comments from Dorothy:

I feel like the FEC Record needs a little more explanation. Maybe “FEC Record news articles inform candidates and committees about FEC developments and are published…”

Under “Members of the Public” – I’d make the last part after the dash its own sentence.

**Comment by emileighoutlaw** [comment]
Made those changes in the content document and in my comment above.

Thanks @AmyKort and @dorothyyeager!
